tests/wpunit/Tribe/Tickets/Commerce/PayPal/__snapshots__/TicketsBlockTest__test_should_render_ticket_block with data set 4__1.php

```diff
@@ -52,15 +52,15 @@ class="tribe-common-b3 tribe-tickets__item__extra__available"
 			<button
 				type="button"
 				class="tribe-common-b3 tribe-tickets__item__details__summary--more"
-				aria-controls="tribe__details__content--13"
+				aria-controls="tribe__details__content--10009"
```

Comment:
I wonder if the updates in ID here are a mistake in any shape or form.

Reply:
I recently updated IDs in the tests for the assertions so that they wouldn't conflict with capacity numbers like 10, 11, etc. There's some caveats to the snapshot assertion allowable differences functionality where it would replace those things.

Reply:
@sc0ttkclark any action I should take here?

Reply:
Nope, nothing further